### PR TITLE
Add 'How to update the volunteer docs' page

### LINKED ISSUE #10: Create "How to update the volunteer docs" page

### PR DESCRIPTION
**Pull request by Adimote**
fixes #10

### CODE REVIEW

File: content/updating-volunteer-docs.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: How to update the volunteer docs
+---
+
+# Github
```
Comment (antonnikitin97) [inline content/updating-volunteer-docs.md]: Git**H**ub #Pedantry

Comment (RealOrangeOne) [reply content/updating-volunteer-docs.md]: Agreed, casing should be proper

**Comment by RealOrangeOne** [comment]
I really think this page is a little pointless. There's a link to the repo in the sidebar. I was expecting this page to detail things like the CI, deploy previews, PRs, reviews, etc etc.

**Comment by RealOrangeOne** [comment]
We should link to one, maintaining one ourselves sounds like a lot of wasted effort

**Comment by Adimote** [comment]
@RealOrangeOne turns out I couldn't find a nice tutorial, I've created my own one.

**Comment by Adimote** [comment]
if #28 is merged, we should note that .spelling files must be alphabetically sorted

File: content/essential-knowledge/updating-volunteer-docs.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+title: How to update the volunteer docs
+---
+
+We have a documentation site on how to volunteer (you're currently on it). Everyone can edit or add to this documentation (even competitors) This page gives you some tips on how to do so.
```
Comment (trickeydan) [inline content/essential-knowledge/updating-volunteer-docs.md]: Second sentence requires some extra punctuation here.

-  Add a comma and remove the capital on `This`
- Or, add a Full Stop.

File: content/essential-knowledge/updating-volunteer-docs.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+## Editing a page in GitHub
+
+Lets say you've spotted some bad grammar on this page, and you want to edit the page to fix it.
```
Comment (trickeydan) [inline content/essential-knowledge/updating-volunteer-docs.md]: s/Lets/Let's/

https://www.grammarly.com/blog/lets-vs-lets/

File: content/essential-knowledge/updating-volunteer-docs.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+These pages are stored in GitHub on the [volunteer-docs repository][volunteer-docs]. This is where you should go to make your changes.
+
+To make your changes you need to be logged in on GitHub, if you don't have an account you should [sign up to GitHub][register-github].
+
+- All content is available in the [content folder][content-dir].
+- All images/extra files are in the [static folder][static].
```
Comment (trickeydan) [inline content/essential-knowledge/updating-volunteer-docs.md]: Minor: Inconsistent link names here, doesn't matter too much

File: content/essential-knowledge/updating-volunteer-docs.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+All pages are stored as [markdown][markdown-ref] files (ones ending in `.md`). Once you've found the file for the page you want to edit  (`updating-volunteer-docs.md` in this case), simply open the file, then click on the little pencil in the top-right corner to start editing.
+
+![Pencil icon to click on for editing](/img/essential-knowledge/updating-docs/pencil.png)
+
+Once you've made your edit, scroll to the bottom and write what you changed in the 'Commit changes' box. Then pick a branch name, this should just be a summary of your changes in a shortened form. Once you've done this, then click the 'Propose file change' button. This will take you to the 'Open a pull request' page, (see the 'Creating a Pull Request' section below).
```
Comment (trickeydan) [inline content/essential-knowledge/updating-volunteer-docs.md]: Can branch names have spaces in?

File: content/essential-knowledge/updating-volunteer-docs.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+There are a few checks which run on the code, what follows is a quick summary of the most common errors and how to fix them:
+
+- **'Spelling errors found':** We have a spell checker, this is great for catching typos, but has quite a high false-positive rate. If you're certain you've spelt the word highlighted in red correctly, you can add it to the [.spelling file][spelling-file] (if using GitHub, use the drop-down in the top left corner to select your branch before you make changes! This means it will update the pull request with your new change.)
+
+- **'N broken' broken links**: We have a checker for broken links, this means one of the links you added was broken, but sometimes means that a site we have previously linked to has just gone down. If you didn't add the link, comment the fact then ignore it.
+
+### Review process
+
+Once you submit a Pull Request, it is then sent for review by us. We will then comment on your changes, or if it seems fine we will approve it. Once it has been approved you can then click on the big green 'Merge pull request' button which will update the main website with your changes!
+
```
Comment (trickeydan) [inline content/essential-knowledge/updating-volunteer-docs.md]: Is it worth mentioning removal of the branch after it is merged?

File: content/essential-knowledge/updating-volunteer-docs.md (hunk 5 above)
Comment (antonnikitin97) [reply content/essential-knowledge/updating-volunteer-docs.md]: I agree with Dan this needs to be mentioned, do we also have a standard convention of naming our branches?

File: content/essential-knowledge/updating-volunteer-docs.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+![Commit Changes form asking for you to fill out the changes to commit](/img/essential-knowledge/updating-docs/commit-changes.png)
+
+## Creating a Pull Request
+
+Once you've clicked the 'Commit Changes' button in GitHub, or if you've just pushed a change manually, you can open a pull request.
+
+A Pull Request is a request for changes to be 'pulled' into the main version of the website (aka the master version).
+
+The text should automatically be filled out for you, so as long as it makes sense, click the 'Create pull request' button. This will request for people to take a look at your changes and review it.
```
Comment (antonnikitin97) [inline content/essential-knowledge/updating-volunteer-docs.md]: maybe mention that you can manually add specific people to review your changes? :)

Comment (Adimote) [reply content/essential-knowledge/updating-volunteer-docs.md]: fixed in 2531ba4dd99161fff1c006c1030a3edeee30d7aa

File: content/essential-knowledge/updating-volunteer-docs.md (hunk 6 above)
Comment (Adimote) [reply content/essential-knowledge/updating-volunteer-docs.md]: fixed in a3a7b4d37cf2fd723d3cefff334486ec04d0d28f

File: content/essential-knowledge/updating-volunteer-docs.md (hunk 2 above)
Comment (Adimote) [reply content/essential-knowledge/updating-volunteer-docs.md]: fixed in 69975522a82ca86fafdcc94297c7a7327ddfa823

File: content/essential-knowledge/updating-volunteer-docs.md (hunk 3 above)
Comment (Adimote) [reply content/essential-knowledge/updating-volunteer-docs.md]: fixed in 69975522a82ca86fafdcc94297c7a7327ddfa823

File: content/essential-knowledge/updating-volunteer-docs.md (hunk 4 above)
Comment (Adimote) [reply content/essential-knowledge/updating-volunteer-docs.md]: fixed in 69975522a82ca86fafdcc94297c7a7327ddfa823

File: content/essential-knowledge/updating-volunteer-docs.md (hunk 5 above)
Comment (Adimote) [reply content/essential-knowledge/updating-volunteer-docs.md]: fixed in 69975522a82ca86fafdcc94297c7a7327ddfa823

File: content/essential-knowledge/updating-volunteer-docs.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+- **'N broken' broken links**: We have a checker for broken links, this means one of the links you added was broken, but sometimes means that a site we have previously linked to has just gone down. If you didn't add the link, comment the fact then ignore it.
+
+### Review process
+
+Once you submit a Pull Request, it is then sent for review by us. If you know a specific person who should take a look, you can add them in the dropdown on the right, otherwise someone will pick it up when they're free. We will then comment on your changes, or if it seems fine we will approve it. Once it has been approved you can then click on the big green 'Merge pull request' button which will update the main website with your changes! After this you should click the 'Delete branch' button to clean up the branch you made.
```
Comment (antonnikitin97) [inline content/essential-knowledge/updating-volunteer-docs.md]: should  'Pull Request' be capitalised?

File: content/essential-knowledge/updating-volunteer-docs.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+To make your changes you need to be logged in on GitHub, if you don't have an account you should [sign up to GitHub][register-github].
+
+- All content is available in the [content folder][content-dir].
+- All images/extra files are in the [static folder][static-dir].
+
+All pages are stored as [markdown][markdown-ref] files (ones ending in `.md`). Once you've found the file for the page you want to edit  (`updating-volunteer-docs.md` in this case), simply open the file, then click on the little pencil in the top-right corner to start editing.
+
+![Pencil icon to click on for editing](/img/essential-knowledge/updating-docs/pencil.png)
+
+Once you've made your edit, scroll to the bottom and write what you changed in the 'Commit changes' box. Then pick a branch name, this should just be a summary of your changes in a shortened form, spaces aren't allowed. Once you've done this, then click the 'Propose file change' button. This will take you to the 'Open a pull request' page, (see the 'Creating a Pull Request' section below).
```
Comment (antonnikitin97) [inline content/essential-knowledge/updating-volunteer-docs.md]: The sentence :
"Once you've done this, **then** click the 'Propose file change' button" doesn't really read well, maybe remove the "then"?

File: content/essential-knowledge/updating-volunteer-docs.md [hunk 10]
```diff
@@ -0,0 +1,72 @@
+---
+title: How to update the volunteer docs
+---
+
+We have a documentation site on how to volunteer (you're currently on it). Everyone can edit or add to this documentation (even competitors), this page gives you some tips on how to do so.
+
+## Editing a page in GitHub
+
+Let's say you've spotted some bad grammar on this page, and you want to edit the page to fix it.
```
Comment (antonnikitin97) [inline content/essential-knowledge/updating-volunteer-docs.md]: #MinorPedantary: you've used the word 'page' twice really close together.

**Comment by RealOrangeOne** [comment]
I _really_ like the images here, makes things much easier to interpret! As theyre specific to this page, It'd be nicer to have them as [page resources](https://gohugo.io/content-management/image-processing/#the-image-page-resource) rather than under a specific directory in `static/`

File: content/essential-knowledge/updating-volunteer-docs.md [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+
+These pages are stored in GitHub on the [volunteer-docs repository][volunteer-docs]. This is where you should go to make your changes.
+
+To make your changes you need to be logged in on GitHub, if you don't have an account you should [sign up to GitHub][register-github].
+
+- All content is available in the [content folder][content-dir].
+- All images/extra files are in the [static folder][static-dir].
+
+All pages are stored as [markdown][markdown-ref] files (ones ending in `.md`). Once you've found the file for the page you want to edit  (`updating-volunteer-docs.md` in this case), simply open the file, then click on the little pencil in the top-right corner to start editing.
```
Comment (RealOrangeOne) [inline content/essential-knowledge/updating-volunteer-docs.md]: I'm not sure we need

File: content/essential-knowledge/updating-volunteer-docs.md [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+### Pull Request checks
+
+There are also a few checks which run when you create a pull request, these show up at the bottom of the page a minute or to after you've made a pull request. click 'Show All checks' to see all of them. These checks include:
+
+- Checking Spelling and links (CircleCI checks)
+- A sample version of the website is generated (Netlify checks)
+
+If the CircleCI check fails, click on the 'Details' link to see what is broke (ask for help in the comments on if you don't know what is broken or how to fix it. Errors tend to be a bit cryptic)
+
+It's a good idea to check what your website looks like by clicking on the 'Details' of the Netlify check. It should take you to a specially generated version of the site with your changes. You can then look for any obvious errors.
```
Comment (RealOrangeOne) [inline content/essential-knowledge/updating-volunteer-docs.md]: > It should take you to

No, it _will_ take you to. Unless the build fails that is, then you go to an error log

Comment (Adimote) [reply content/essential-knowledge/updating-volunteer-docs.md]: fixed in 9be7417999b06b77c3d102ab0f846e6cc6e644a2

File: content/essential-knowledge/updating-volunteer-docs.md (hunk 10 above)
Comment (Adimote) [reply content/essential-knowledge/updating-volunteer-docs.md]: fixed in

File: content/essential-knowledge/updating-volunteer-docs.md (hunk 9 above)
Comment (Adimote) [reply content/essential-knowledge/updating-volunteer-docs.md]: fixed in 9eb39baca397d377f5bd6e2723a6afd43d85d750

File: content/essential-knowledge/updating-volunteer-docs.md (hunk 8 above)
Comment (Adimote) [reply content/essential-knowledge/updating-volunteer-docs.md]: fixed in 8d99833ca52a56d547431e0ad801641d23ddb0a8

**Pull request merged**